src/Umbraco.Core/Services/ContentPublishingService.cs

```diff
@@ -217,6 +217,25 @@ public async Task<Attempt<ContentPublishingOperationStatus>> UnpublishAsync(Guid
             : Attempt.Fail(ToContentPublishingOperationStatus(result));
     }
 
+    /// <inheritdoc/>
+    public async Task<Attempt<ContentPublishingOperationStatus>> UnpublishMultipleCulturesAsync(Guid key, IEnumerable<string> cultures, Guid userKey)
```

Comment:
I guess it should fail if cultures is empty.. Also it should be a set, as duplicates would be weird